hack/windows/Start-AntreaAgent.ps1

```diff
@@ -76,6 +76,7 @@ if ($LastExitCode) {
 }
 
 if ($StartKubeProxy) {
+    Write-Host "Running Antrea with kube-proxy is no longer supported, this parameter will be removed soon"
```

Comment:
@XinShuYang just a question, will it break Antrea functions if Antrea is running with kube-proxy on Windows? do we record any helpful information about the side-effect?

Reply:
@luolanzone  Starting with Kubernetes 1.26, Antrea no longer supports kube-proxy on Windows due to the deprecation of the userspace datapath. We have not found a way to support the new kernel datapath that kube-proxy uses. As a result, from version 1.26 onwards, Antrea can only operate in proxy-all mode on Windows. For more details, please refer to the documentation at https://github.com/antrea-io/antrea/blob/main/docs/windows.md#add-windows-kube-proxy-daemonset-only-for-kubernetes-versions-prior-to-126.